lstchain/io/lstcontainers.py

```diff
@@ -38,7 +38,6 @@ class DL1ParametersContainer(Container):
     disp_angle = Field(None, 'disp_angle [rad]', unit=u.rad)
     disp_sign = Field(None, 'disp_sign')
     disp_miss = Field(None, 'disp_miss [m]', unit=u.m)
-    hadroness = Field(None, 'hadroness')
```

Comment:
not that it matters that much, but up to know, according to the code of @misabelber, we were using `gammaness` instead of `hadronness`, did you decide to go for hadronness in the last DL1 data format call?

Reply:
If @misabelber could confirm but it seemed to me that we decided that:
- we would use the particle type from simtel to indentify them
- we would use gammaness that is more broadly used today

Reply:
Yes, that's correct!